ui/app/templates/components/database-role-edit.hbs

```diff
@@ -63,6 +63,14 @@
           @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}}
           @value={{stringify (get @model attr.name)}}
         />
+      {{else if (eq attr.options.editType "ttl")}}
```

Comment:
just checking that the editType here doesn't need to be default_ttl?

Reply:
Nope, the `attr.name` is `default_ttl`, but here I'm checking if editType (as set in the options on the model) is ttl, as it is for max_ttl as well, so that we can format it correctly 👍